apps/docs/src/app/platform/component-docs/platform-standard-list-item/platform-standard-list-item-examples/platform-standard-list-item-example.component.html

```diff
@@ -0,0 +1,22 @@
+<h3>Cozy Mode</h3>
+<fdp-list [hasByLine]="true" [navigated]="true">
+    <fdp-list-group-header>Group header 1</fdp-list-group-header>
+    <fdp-standard-list-item title="List item 1" [avatarSrc]="'https://placeimg.com/400/400/nature'"
+        [avatarTitle]="'nature'" [description]="'First text item in Byline (Standard text item)'">
```

Comment:
unnecessary bindings
plz check the whole file

Reply:
updated